page-objects/pages/administrationPages/ADMNSTRTNAdministerStudentsPage.js

```diff
@@ -18,13 +18,23 @@ const addStudentSubmitBtn = 'div.modal.fade.add-modal.in button.btn-submit';
 const passwordInput = '#passwd';
 const createBirthday = '#birthday';
 const sendConsentFormEmails = '.btn-send-links-emails';
+const editStudentBtn = "a[title='Nutzer bearbeiten']";
+const pageTitle = '#page-title';
 
-//
 async function clickAddStudentBtn() {
 	await waitHelpers.waitUntilAjaxIsFinished();
 	await elementHelpers.clickAndWait(addStudentBtn);
 }
 
+async function clickEditStudentBtn() {
+	// to make it work on new admin tables
+	try {
+		await elementHelpers.click("a[datatest-id='edit_student_button']");
```

Comment:
move it to -  example const variableName = 'a[datatest-id='edit_student_button']' and the use "variableName" in function, use single quotes